tldr.py

```diff
@@ -504,6 +508,40 @@ def main() -> None:
                 with open(command, encoding='utf-8') as open_file:
                     output(open_file.read().encode('utf-8').splitlines(),
                            plain=options.markdown)
+    elif options.search:
+        command = '-'.join(options.command)
+        get_commands(options.platform)
+        page = ''
```

Comment:
```suggestion
        page = None
```

I think this makes the intention clearer.